client/analytics/report/index.js

```diff
@@ -94,6 +94,19 @@ export const getReports = () => {
 	return applyFilters( REPORTS_FILTER, reports );
 };
 
+/**
+ * The Customers Report will not have the `report` param supplied by the router/
+ * because it no longer exists under the path `/analytics/:report`. Use `props.path`/
+ * instead to determine if the Customers Report is being rendered.
+ *
+ * @param {Object} params - url parameters
+ * @param {string} path - url path
+ * @return {string} - report parameter
+ */
+const getReportParam = ( params, path ) => {
```

Comment:
```suggestion
const getReportParam = ( { params, path } ) => {
```
Reports are breaking due to this 😄